savant/tests/test_basic.py

```diff
@@ -0,0 +1,19 @@
+import unittest
```

Comment:
This needs to go into a new Pull Request. Please target your PRs at only a single github issue.

For now, please take this out, and open a new PR.